deploy/cmd/create.go

```diff
@@ -75,6 +76,15 @@ func createCmd() *cobra.Command {
 				return err
 			}
 
+			if _, err = cutils.LoadCertificateForAccount(clientCtx, clientCtx.Keyring); err != nil {
+				if os.IsNotExist(err) {
```

Comment:
Shouldn't we also check that the certificate is on chain?

Reply:
💯

Reply:
fixed